modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -41,19 +41,24 @@
 	if(C.mob_biotypes & MOB_ROBOTIC)
 		C.nutrition = min(C.nutrition + 5, NUTRITION_LEVEL_FULL-1)
 	..()
+
 // Catnip
 /datum/reagent/pax/catnip
 	name = "Catnip"
 	taste_description = "grass"
 	description = "A colourless liquid that makes people more peaceful and felines happier."
 	metabolization_rate = 1.75 * REAGENTS_METABOLISM
 
-/datum/reagent/pax/catnip/on_mob_life(mob/living/carbon/M)
-	if(isfelinid(M) || istajaran(M))
-		if(prob(20))
-			M.emote("nya")
-		if(prob(20))
-			to_chat(M, span_notice("[pick("Headpats feel nice.", "Backrubs would be nice.", "Mew")]"))
-	else
-		to_chat(M, span_notice("[pick("I feel oddly calm.", "I feel relaxed.", "Mew?")]"))
+/datum/mood_event/catnip
+	description = span_nicegreen("That plant has left me feeling great!\n")
+	mood_change = 1
+	timeout = 4 MINUTES
+
+/datum/reagent/pax/catnip/on_mob_metabolize(mob/living/affected_mob)
+	. = ..()
+	SEND_SIGNAL(affected_mob, COMSIG_ADD_MOOD_EVENT, "catnip", /datum/mood_event/catnip)
+
+/datum/reagent/pax/catnip/on_mob_life(mob/living/carbon/affected_carbon)
+	if(prob(10))
```

Comment:
you forgot the species check. Also undocumented change that you lowered the prob chance (also keep the emote to just nya please)